example/deployment.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     spec:
       containers:
       - name: nats-operator
-        image: nats-io/nats-operator:latest
+        image: wallyqs/nats-operator:0.2.0-v1beta1
```

Comment:
This doesn't look right to me.

Reply:
@pires , Great catch, thank you for noticing.  I asked @wallyqs to do this temporarily - Docker takes a *long* time to approve new images, and we'd like the community to be able to experiment with this in beta without having to build and use their own images for the operator. In the README, we should document that this image is used in the beta period, and only until the official docker image is available.  Note that for GA (v0.1.0) we'll be sure to use an official docker image.  We're certainly open to any other suggestions...